digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/masstransitstop/MassTransitStopOperations.scala

```diff
@@ -24,6 +24,12 @@ object BusStopType {
   case object Unknown extends BusStopType { def value = 99 }
 }
 
+object BusStopAdministrator {
```

Comment:
tätä ei vai ollut määritetty missään muualla ennen? mä kyl muistelisin että jokin tälläin olisi jo olemassa. @MJKaakinen

Reply:
hyym ei varmaan ollut

Reply:
Koetettiin Antin kanssa etsiä muttei löydetty.

Reply:
löytyy riviltä 46 alkaen